examples/apps/tree-comparison/src/model/newTreeInventoryList.ts

```diff
@@ -190,11 +196,9 @@ export class NewTreeInventoryList extends DataObject implements IInventoryList {
 			}
 
 			// Search for deleted inventory items to update our collection
-			const currentInventoryIds = [...this.inventory.inventoryItems].map(
-				(inventoryItemNode) => {
-					return inventoryItemNode.id;
-				},
-			);
+			const currentInventoryIds = [...this.inventoryItemList].map((inventoryItemNode) => {
```

Comment:
You should now be able to do `this.inventoryItemList.map(..)` without first copying into an array, since SharedListNode<T> extends 'readonly T[]'.